gnomad_qc/v4/resources/sample_qc_def_for_edit.json

```diff
@@ -0,0 +1,2256 @@
+                        },
+                        "min_site_callrate": {
+                            "definition": "Filtering threshold used for sample callrate computed on only predetermined QC variants (predetermined using CCDG genomes/exomes, gnomAD v3.1 genomes, and UKB exomes) after ADJ filtering."
+                        }
+                    }
+                },
+                "bi_allelic_sample_qc_parameters": {
+                    "definition": "Parameters used to compute sample QC metrics on bi-allelic SNVs.",
+                    "subfields": {
+                        "gq_bins": {
+                            "definition": "Tuple containing cutoffs for genotype quality (GQ) scores."
+                        },
+                        "dp_bins": {
+                            "definition": "Tuple containing cutoffs for depth (DP) scores."
+                        }
+                    }
+                }
+            }
+        },
+        "population_inference_parameters": {
+            "definition": "Parameters used to assign ancestry groups.",
+            "subfields": {
+                "assign_pops_from_pc_params": {
+                    "definition": "Parameters used in a random forest model to assign ancestry group labels based on the results of PCA.",
+                    "subfields": {
+                        "min_assignment_prob": {
+                            "definition": "Minimum probability of belonging to a given ancestry group for the ancestry group to be set (otherwise set to 'remaining')."
+                        },
+                        "error_rate": {
+                            "definition": "Random forests error rate on the training data for acestry group assignment."
+                        }
+                    }
+                },
+                "min_prob": {
+                    "definition": "Minimum probability of belonging to a given ancestry group for the ancestry group to be set (otherwise set to 'remaining')."
+                },
+                "include_unreleasable_samples": {
+                    "definition": "Whether unreleasable samples were included in PCA."
+                },
+                "pcs": {
+                    "definition": "List of PCs used in the RF."
+                },
+                "include_v2_known_in_training": {
+                    "definition": "Whether v2 known populations were included in the RF training data."
+                },
+                "v3_population_spike": {
+                    "definition": "List of v3 populations spiked into the RF training data."
+                },
+                "v4_population_spike": {
+                    "definition": "List of v4 populations spiked into the RF training data."
+                },
+                "min_prob_cutoffs": {
+                    "definition": "The minimum probability of belonging to a given ancestry group for the ancestry group to be set (otherwise set to `Remaining`)."
```

Comment:
```suggestion
                    "definition": "The minimum probability of belonging to a given ancestry group for the ancestry group to be set (otherwise set to 'remaining')."
```